### PR TITLE
Add more documentation on how to enable/disable BBS features

### PR DESCRIPTION
**Pull request by mrnugget**
This adds more documentation to answer questions we anticipate might come up in "failure modes". See: https://github.com/sourcegraph/customer/issues/18

### CODE REVIEW

File: doc/integration/bitbucket_server.md [hunk 1]
```diff
@@ -48,17 +48,25 @@ In order to update the plugin, follow the same steps as for installing it, which
...
+
+To disable native code intelligence, simply set **Sourcegraph URL** to an empty value. Note that this will also disable [Webhooks](#webhooks)!
+
 ### Webhooks
 
 Once the plugin is installed, go to **Administration > Add-ons > Sourcegraph** to see a list of all configured webhooks and to create a new one.
 
 To configure a webhook on the Sourcegraph side, set the [`"plugin.webhooks"` property in the Bitbucket Server configuration](../admin/external_service/bitbucket_server.md#webhooks). Once that is configured Sourcegraph automatically makes sure in the background that a global webhook for usage with [Campaigns](../user/campaigns.md) is created on the Bitbucket Server instance.
 
+Disabling the webhooks is as easy as removing the `"plugin.webhooks"` property and deleting the webhook pointing to your Sourcegraph instance under **Administration > Add-ons > Sourcegraph**.
```
Comment (eseliger) [inline doc/integration/bitbucket_server.md]: Maybe this could be an automated process in the future?

File: doc/integration/bitbucket_server.md [hunk 2]
```diff
@@ -48,17 +48,25 @@ In order to update the plugin, follow the same steps as for installing it, which
...
+
 ### Fast permission syncing
 
 The plugin also supports an optional method of faster ACL permissions syncing that aims to improve the speed of fetching a user's permissions from Bitbucket (which can reduce the time a user has to wait to run a search if their permissions data has expired).
 
-You can enable this feature when [configuring the connection to your Bitbucket Server instance on Sourcegraph](../admin/external_service/bitbucket_server.md#repository-permissions).
+You can enable this feature when [configuring the connection to your Bitbucket Server instance on Sourcegraph](../admin/external_service/bitbucket_server.md#repository-permissions). For more information on when permissions are fetched, how long they're cached and how to configure that behaviour see our documentation on [Repository permissions](../admin/repo/permissions.md).
```
Comment (eseliger) [inline doc/integration/bitbucket_server.md]:
```suggestion
You can enable this feature when [configuring the connection to your Bitbucket Server instance on Sourcegraph](../admin/external_service/bitbucket_server.md#repository-permissions). For more information on when permissions are fetched, how long they're cached and how to configure that behavior see our documentation on [Repository permissions](../admin/repo/permissions.md).
```

As per https://about.sourcegraph.com/handbook/communication/style_guide

File: doc/integration/bitbucket_server.md (hunk 1 above)
Comment (mrnugget) [reply doc/integration/bitbucket_server.md]: Yeah, that's a great idea.

Comment (mrnugget) [reply doc/integration/bitbucket_server.md]: Created a ticket for it: https://github.com/sourcegraph/sourcegraph/issues/8914

File: doc/integration/bitbucket_server.md [hunk 3]
```diff
@@ -48,17 +48,25 @@ In order to update the plugin, follow the same steps as for installing it, which
 
 When the Sourcegraph instance connected to the Bitbucket Server plugin is updated, so will the code that's fetched by the plugin to enable native code intelligence. No manual steps required. (See the [Technical Details](#technical-details) section on how this works.)
 
+### Native code intelligence
+
+Once the plugin is installed and the **Sourcegraph URL** is set under **Administration > Add-ons > Sourcegraph** native code intelligence is enabled when browsing code or pull requests on your Bitbucket Server installed.
```
Comment (ryan-blunden) [inline doc/integration/bitbucket_server.md]:
```suggestion
Once the plugin is installed and the **Sourcegraph URL** is set under **Administration > Add-ons > Sourcegraph**, native code intelligence is enabled when browsing code or pull requests on your Bitbucket Server instance.
```

File: doc/admin/repo/permissions.md [hunk 4]
```diff
@@ -165,11 +165,15 @@ Go to your Sourcegraph's *Manage repositories* page (i.e. `https://sourcegraph.e
...
 
 The default `hardTTL` is **3 days**, after which a user's cached permissions must be updated before any user action can be authorized. While the update is happening an error is returned to the user. The default `hardTTL` value was chosen so that it reduces the chances of users being forced to wait for their permissions to be updated after a weekend of inactivity.
 
-There is also an experimental feature that allows for faster permissions fetching that can be enabled via the [Bitbucket Server plugin](../../../integration/bitbucket_server.md).
+### Fast permission sync with Bitbucket Server plugin
+
+By installing the [Bitbucket Server plugin](../../../integration/bitbucket_server.md) you can make use of the fast permission sync feature that allows using Bitbucket Server permissions on larger instances.
```
Comment (ryan-blunden) [inline doc/admin/repo/permissions.md]:
```suggestion
By installing the [Bitbucket Server plugin](../../../integration/bitbucket_server.md), you can make use of the fast permission sync feature that allows using Bitbucket Server permissions on larger instances.
```

File: doc/integration/bitbucket_server.md (hunk 1 above)
Comment (ryan-blunden) [inline doc/integration/bitbucket_server.md]:
```suggestion
Disabling the webhook is as easy as removing the `"plugin.webhooks"` property and deleting the webhook pointing to your Sourcegraph instance under **Administration > Add-ons > Sourcegraph**.
```

File: doc/integration/bitbucket_server.md [hunk 5]
```diff
@@ -48,17 +48,25 @@ In order to update the plugin, follow the same steps as for installing it, which
...
 
+Disabling the webhooks is as easy as removing the `"plugin.webhooks"` property and deleting the webhook pointing to your Sourcegraph instance under **Administration > Add-ons > Sourcegraph**.
+
 ### Fast permission syncing
 
 The plugin also supports an optional method of faster ACL permissions syncing that aims to improve the speed of fetching a user's permissions from Bitbucket (which can reduce the time a user has to wait to run a search if their permissions data has expired).
 
-You can enable this feature when [configuring the connection to your Bitbucket Server instance on Sourcegraph](../admin/external_service/bitbucket_server.md#repository-permissions).
+You can enable this feature when [configuring the connection to your Bitbucket Server instance on Sourcegraph](../admin/external_service/bitbucket_server.md#repository-permissions). For more information on when permissions are fetched, how long they're cached and how to configure that behavior see our documentation on [Repository permissions](../admin/repo/permissions.md).
```
Comment (ryan-blunden) [inline doc/integration/bitbucket_server.md]:
```suggestion
You can enable this feature when [configuring the connection to your Bitbucket Server instance on Sourcegraph](../admin/external_service/bitbucket_server.md#repository-permissions). For more information on when permissions are fetched, how long they're cached and how to configure that behavior, see our documentation on [Repository permissions](../admin/repo/permissions.md).
```

**Pull request merged**